Build/exec.sh

```diff
@@ -32,11 +32,23 @@ then
 		time bitbake mistysom-image
 		echo "copying compiled images into 'out/'"
 		cp -r $WORK/build/tmp/deploy/images/ ${OUTDIR}
+		cd ${OUTDIR}/images/smarc-rzv2l
+		rm bl*
+		rm fip*
+		rm Flash_Writer*
+		wget https://github.com/MistySOM/wiki/blob/master/files/bootloader/rzv2l/Flash_Writer_SCIF_RZV2L.mot
+		wget https://github.com/MistySOM/wiki/blob/master/files/bootloader/rzv2l/bl2_bp-MistySOMV2L.srec
+		wget https://github.com/MistySOM/wiki/blob/master/files/bootloader/rzv2l/fip-MistySOMV2L.srec
 	else
 		time sh -c "bitbake mistysom-image && bitbake mistysom-image -c populate_sdk"
 		echo "copying compiled images & SDK directories into 'out/'"
 		cp -r $WORK/build/tmp/deploy/sdk/ ${OUTDIR}
 		cp -r $WORK/build/tmp/deploy/images/ ${OUTDIR}
+		cd ${OUTDIR}
+		rm smarc-rzv2l/bl2*
+		rm smarc-rzv2l/fip*
```

Comment:
Why the lines above are different compared to the ones in the other condition?
Why is it missing the Flash_Writer?
why are you downloading in another directory compared to the other condition?

Reply:
> Why the lines above are different compared to the ones in the other condition?
Because it's a draft and I haven't done a full review, will fix!
> Why is it missing the Flash_Writer?
pending reviw, will fix!
> why are you downloading in another directory compared to the other condition?
pending review, will fix!